### ISSUE TITLE
Use babel-preset-env

### ISSUE THREAD

**Issue by rolandjitsu** [monounity/karma-typescript-es6-transform#4]
I'm seeing the following warning from npm when I use this package:
```text
warning karma-typescript-es6-transform > babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update!
```

**Comment by erikbarke** [comment]
Ok, it's time to update babel!

**Comment by erikbarke** [comment]
I've upgraded to `babel-preset-env` now, the fix is in #5 😌

**Comment by erikbarke** [comment]
1.0.3 is on npm now!
